### DISCUSSION TITLE
GO ANALYSIS

### DISCUSSION THREAD

**Discussion by alexsanr1** [geneontology/helpdesk#463] [Q&A]
Hello, I have a collection of N mutants and I have analysed a certain phenotype in this collection, thus obtaining X candidates. I would like to know if the GO enrichment analysis has to be done taking into account the whole genome of the organism or only on the collection of N mutants that I have analysed.

**Comment by suzialeksander** [comment]
Hi @alexsanr1, 

The background set you choose usually reflects your experiment.  While you can use the entire genome, or at least the entire annotated genome for many organisms, this may result in results that are too general. You might get more helpful results if you narrow the background down to the genes it was possible to detect in your assay.
